lib/permutations.rb

```diff
@@ -1,4 +1,25 @@
+#Write a method which will take two strings as arguments and returns true if one string is a permutation of the other.
+#Time complexity: O(n+m)
+#Space complexity: O(n)
 
 def permutations?(string1, string2)
```

Comment:
Very close, since both are, presumed, to be the same length the time complexity is O(n)